requirements/base.in

```diff
@@ -8,6 +8,7 @@ django-model-utils
 django-rest-swagger
 django-waffle
 djangorestframework
+edx-api-doc-tools
```

Comment:
Do you need this requirement? I don't see this library in use in this pull request. If not, please remove it and update the requirements files.

Reply:
Oh i don't think so, updated 👍